### PR TITLE
fix tf code to match default subnets

### PR DESCRIPTION
**Pull request by Bharathkumarraju**

### CODE REVIEW

File: eks/data.tf [hunk 1]
```diff
@@ -27,14 +17,12 @@ data "aws_subnets" "public" {
   filter {
     name = "availability-zone"
     values = [
-      "${var.aws_region}a",
-      "${var.aws_region}b",
-      "${var.aws_region}c"
+      "${var.aws_region}c",
+      "${var.aws_region}d"
```
Comment (by-nelson) [inline eks/data.tf:21]: I don't think this is the case. In the labs I get 6 subnets (a-f)

**Comment by Bharathkumarraju** [comment]
Yup right for me it get others

**Comment by Bharathkumarraju** [comment]
Plz ingnore then thanks

**Comment by fireflycons** [comment]
Hi @Bharathkumarraju @by-nelson 

The terraform code is currently being updated to handle a number of edge cases, such as there not being sufficient subnets to deploy a cluster (intermittent lab issue), and for recent EKS changes released at re:invent in November. 
Comments in here have been noted.
Thanks

**Pull request closed without merging**